specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-02-01/examples/AgentPoolsCreate_Spot.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
new parameters are added into  a new file( `AgentPoolsCreate_Spot.json`) other than `AgentPoolsCreate_Update.json`, I am not sure whether this is the right way.
@zikalino could you take a look? Thanks.

Reply:
It's reviewed earlier. https://github.com/Azure/azure-rest-api-specs/pull/8342

Reply:
this is just the example spec and its linked to [here](https://github.com/Azure/azure-rest-api-specs/pull/8388/files#diff-f06d14f220960f1582583c23b31af698R724)